.prettierrc.json

```diff
@@ -3,5 +3,13 @@
   "tabWidth": 2,
   "useTabs": false,
   "semi": true,
-  "singleQuote": true
+  "singleQuote": true,
+  "overrides": [
+    {
+      "files": "*.svg",
+      "options": {
+        "parser": "html"
+      }
+    }
+  ]
```

Comment:
Had to be done to be able to parse the `.svg` file. Not sure why the pre-commit did not stop me the last time I added an SVG. We previously used the `@prettier/plugin-xml` for this, but I couldn't get it to work, and this seems like a better way anyways 🤷 

Will create a PR for removal of `@rettier/plugin-xml`